tests/unit/backend/corpora/common/utils/test_dropbox.py

```diff
@@ -65,18 +65,18 @@ def test__get_file_info__OK(self):
         postive_test.file_info()
 
     def test__get_file_info__neg(self):
-        with self.subTest("404"):
+        with self.subTest("Missing Headers"):
             negative_test = from_url("https://www.dropbox.com/s/12345678901234/test.h5ad?dl=1")
```

Comment:
swap fabricated "missing headers" test for a fabricated 404 (below) in case Dropbox switches back to returning 404.